src/main/java/com/javadiscord/javabot/commands/reaction_roles/HelpRole.java

```diff
@@ -0,0 +1,62 @@
+package com.javadiscord.javabot.commands.reaction_roles;
+
+import net.dv8tion.jda.api.entities.ChannelType;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
+import net.dv8tion.jda.api.events.message.react.MessageReactionRemoveEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class HelpRole  extends ListenerAdapter {
+
+    @Override
+    public void onMessageReactionAdd(MessageReactionAddEvent event) {
+
+        if (event.getChannelType() == ChannelType.TEXT) {
+            if (!event.getUser().isBot()) {
+                long guildid = event.getGuild().getIdLong();
+                long channelid = event.getChannel().getIdLong();
+                long messageid = event.getMessageIdLong();
+                String emote = "";
+                if (guildid == 883199663891685446L && channelid == 883199663891685448L && messageid == 888696596303327263L) {
+                    if (event.getReactionEmote().isEmoji()) {
+                        emote = event.getReactionEmote().getEmoji();
+                        System.out.println(emote);
+
+                        if(emote.equals("✅")) {
+                            Guild guild = event.getGuild();
+                            guild.addRoleToMember(event.getMember(), guild.getRoleById(888699748159193089L)).queue();
+                        }}
+
+                }
```

Comment:
- Remove any System.out.println calls. If you need to log things, use the SLF4J logger.
- Add a space on line 25 between `if` and the opening parenthesis.
- Use camelCase for variable names.
- Remove hard-coded discord entity ids. The guild id is not necessary at all, since we assume that every guild the bot is in should support this functionality. The channel, message, and role ids would probably be best put into `com.javadiscord.javabot.properties.config.guild.ModerationConfig`. Take note of the existing properties for that config object, and add your own role/channel/message id properties, and the associated helper methods to get the actual role/channel/message using the config's guild context.
- I would also recommend putting the emoji into the config as a String, or even a List<String> so that we can support multiple possible emojis.